fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/filter/LoanCOBFilterHelper.java

```diff
@@ -259,4 +261,10 @@ private List<Long> getLoanIdList(String pathInfo) {
     public void executeInlineCob(List<Long> loanIds) {
         inlineLoanCOBExecutorService.execute(loanIds, JOB_NAME);
     }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        objectMapper.configure(JsonReadFeature.ALLOW_UNESCAPED_CONTROL_CHARS.mappedFeature(), true);
```

Comment:
![image](https://github.com/apache/fineract/assets/5830589/1f9c5052-747f-4c96-8bd5-592f3be7e0d7)

Reply:
This feature is deprecated and this goes against JSON Spec. In long rung it's advised to fix it on the client side to send proper valid JSON requests.